Makefile

```diff
@@ -84,12 +85,10 @@ tidy:
 check: fmt lint check-static tidy
 
 coverage:
-	GO111MODULE=off go get github.com/wadey/gocovmerge
-	gocovmerge "$(TEST_DIR)"/cov.* | grep -vE ".*.pb.go" > "$(TEST_DIR)/tmp_unit_cov.out"
-	grep -v "$(CDC_PKG)/cdc/kv/testing.go" "$(TEST_DIR)/tmp_unit_cov.out" >"$(TEST_DIR)/unit_cov.out"
+	GO111MODULE=off go get github.com/zhouqiang-cl/gocovmerge
```

Comment:
what's the different with the origin version

Reply:
The new version fixes a bug when merging multiple coverage files (https://github.com/zhouqiang-cl/gocovmerge/commit/5256314471af44fe8bccbf0a1db1dd1f85637c1a)